core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -113,7 +115,7 @@ class DAGScheduler(
   private val failedEpoch = new HashMap[String, Long]
 
   private val dagSchedulerActorSupervisor =
-    env.actorSystem.actorOf(Props(new DAGSchedulerActorSupervisor(this)))
+    env.actorSystem.actorOf(Props(new DAGSchedulerActorSupervisor(this)), "DAGSchedulerSupervisor")
```

Comment:
OutputCommitCoordinator can be an actor that is architected similarly to MapOutputTracker's actor. DAGScheduler and SparkHadoopWriter will both send messages to this actor but the actor will live on the driver side. I'll implement this.